js/main.js

```diff
@@ -0,0 +1,38 @@
+/*
+Случайное целое число из переданного диапазона включительно. Источник: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+*/
+
+function randomInteger(min, max) {
+  if (min < 0){
+    return ('Ошибка: диапазон может быть только положительный');
+  }
+  if (max <= min){
+    return ('Ошибка: значение ДО не может быть меньше или равно ОТ');
+  }
+  /*случайное целое от min до (max+1), так как в Math.random max не включительно*/
+  min = Math.ceil(min);
+  max = Math.floor(max);
+  const randNum = min + Math.random() * (max + 1 - min);
+  /*
+  если использовать Math.round() для округления, то это может сделать распределение неравномерным
+  Math.floor() - округление вниз. Округляет аргумент до ближайшего меньшего целого.
+  */
+  return Math.floor(randNum);
+}
+/*число от 1  ... до 1.9999999999  округлится до 1
+число от 4  ... до 4.9999999999  округлится до 4
+все интервалы имеют одинаковую длину, что выравнивает вероятность получения случайных чисел
+*/
+randomInteger(1.9999999999, 4.9999999999);
+
+/* Проверка максимальной длины строки
+Источник: https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/String/length */
+
+function checkStrLength(string, maxLength) {
+  if (string === null || string.length <= maxLength) {
```

Comment:
Можно упростить избыточность, результат логических операций чаще всего или уже имеет тип boolean (для сравнений и && или !), либо подразумевается
```suggestion
    return string.length <= maxLength
```
Проверка `string === null` имеет место быть, но, если хочешь проверять аргументы, лучше проверь на тип, число ещё можно проверить на то, чтобы оно было натуральным, функция не имеет смысла с отрицательным или нулевым значением длины
```suggestion
     if (typeof string !==  'string') {
      throw new TypeError('Function param string is not a string!')
     }
     if (typeof maxLength !== 'number' || maxLength < 1) {
      throw new TypeError('Function param maxLength is not a natural number!')
     }

    return string.length <= maxLength
```